<launch>
    <include file="$(find lightweight_human_pose_estimation)/launch/camera.launch" >
    </include>

    <include file="$(find lightweight_human_pose_estimation)/launch/human_pose_estimation.launch" >
        <arg name ="checkpoint_path"        value="$(find lightweight_human_pose_estimation)/script/checkpoints/checkpoint_iter_370000.pth"/>
        <arg name ="height_size"            value="256"/>
        <arg name ="pose_img_show_flag"     value="true"/>
        <arg name ="pose_pub_result_image"  value="true"/>
        <arg name ="needs_time_stamp"       value="true"/>
        <!-- <arg name ="pose_image_topic_name"  value="/camera/rgb/image_raw"/>  -->
        <arg name ="pose_image_topic_name"  value="/rgb/image_raw"/> 
        <arg name ="cpu"                    value="true"/>
        <arg name ="track"                  value="1"/>
        <arg name ="smooth"                 value="1"/>
    </include>
    <node pkg="rqt_image_view" type="rqt_image_view" name="image_view_debug" args="/human_pose_estimation/detect_result"/>
</launch>